test/unit/three_scale/swagger/specification_test.rb

```diff
@@ -57,12 +72,11 @@ class ThreeScale::Swagger::SpecificationTest < ActiveSupport::TestCase
 
   private
 
+  def load_fixture file
+    File.read Rails.root.join "test", "fixtures", "swagger", "#{file}.json"
```

Comment:
We can rely on the defined `fixtures_path` and use https://github.com/3scale/porta/blob/2ef2dc9b3addb1b75df2273530a0a24443272b72/test/unit/three_scale/policies/specification_test.rb#L8

Reply:
I didn't really touch this, but good opportunity to improve it. Thanks.